spec/controllers/hyrax/file_sets_controller_spec.rb

```diff
@@ -68,8 +68,14 @@
     end
 
     describe "#update" do
+      let(:parent) do
+        create(:work, :public, user: user)
+      end
       let(:file_set) do
-        create(:file_set, user: user)
+        create(:file_set, user: user).tap do |file_set|
```

Comment:
instead of explicitly creating the `parent` and manually saving the fileset as a member, could we use the `:file_with_work` factory?

```ruby
let(:file_set) do
  create(:file_with_work, user: user)
end
```

?

Reply:
Same question for instances of this below.

Reply:
On your recommendation I tried this, but it doesn't add the file set to the ordered members.

Reply:
The create file with work does not seem to add the file set to the works ordered members.